modules/unsupported/mbtiles/src/main/java/org/geotools/mbtiles/MBTilesMetadata.java

```diff
@@ -1,17 +1,48 @@
 package org.geotools.mbtiles;
+import java.util.Arrays;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 import org.geotools.geometry.jts.ReferencedEnvelope;
+import org.geotools.referencing.CRS;
+import org.geotools.util.logging.Logging;
 import org.opengis.geometry.Envelope;
+import org.opengis.geometry.MismatchedDimensionException;
+import org.opengis.referencing.FactoryException;
 
 public class MBTilesMetadata {
     
+    private static Logger LOGGER = Logging.getLogger(MBTilesMetadata.class); 
+    
     protected static Pattern patternEnvelope = Pattern.compile(" *(\\-?[0-9\\.]*) *, *(\\-?[0-9\\.]*) *, *(\\-?[0-9\\.]*) *, *(\\-?[0-9\\.]*) *");
     
-    public enum t_type {OVERLAY, BASE_LAYER} ;
+    public enum t_type {
+        OVERLAY("overlay"), 
+        BASE_LAYER("baselayer");
```

Comment:
Underscore in the enum, but not in the identifier? I assume this is inteded, but just want to verify.